loader/loader.c

```diff
@@ -5995,7 +6013,7 @@ VkResult setup_loader_term_phys_devs(struct loader_instance *inst) {
     // For each ICD, query the number of physical devices, and then get an
     // internal value for those physical devices.
     icd_term = inst->icd_terms;
-    for (uint32_t icd_idx = 0; NULL != icd_term; icd_term = icd_term->next, icd_idx++) {
+    for (; NULL != icd_term; icd_term = icd_term->next, icd_idx++) {
         icd_phys_dev_array[icd_idx].count = 0;
         icd_phys_dev_array[icd_idx].phys_devs = NULL;
         icd_phys_dev_array[icd_idx].this_icd_term = NULL;
```

Comment:
This could be a memset, right?

Reply:
Ugh, I actually memset the whole structure above.  Stripping this out.